hoon/lib/jock.hoon

```diff
@@ -1016,32 +1106,32 @@
   ++  cons
     |=  q=jype
     ^-  jype
-    [t q]^%$
+    [jyp q]^%$
   ::
   ++  unify
     |=  v=jype
     ^-  (unit jype)
-    ?^  -<.t
-      ?@  -<.v
-        ?:  =(%untyped -.p.v)
-          `t
+    ?^  -.-.jyp
```

Comment:
please use consistent -< syntax instead of -.-

Reply:
There are cases in which `->.tokens` fails and `-.+.tokens` succeeds a type check.

Reply:
Joe says this shouldn't be the case, so I was probably changing two things instead of one when I ran into it.  I'll investigate and adjust appropriately.